examples/xendit-example.php

```diff
@@ -0,0 +1,155 @@
+<?php
```

Comment:
Could you please remove this file and instead add a `README.md` document in the src/Drivers/Xendit?

I merge this PR, please do this refactor in a seperate PR.